### PR TITLE
Update text for Plugfest and Testing

### PR DESCRIPTION
**Pull request by sebastiankb**
solves #123

### CODE REVIEW

File: wot-ig-2023-draft.html [hunk 1]
```diff
@@ -284,20 +284,25 @@ <h2>Scope</h2>
...
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
+	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Intersted Group supports the realization of 
+	 the testing phase and, similar to PlugFest, helps with the inclusion of implementations, e.g. from open source projects and partners.
```
Comment (mmccool) [inline wot-ig-2023-draft.html:303]: Somewhere we should state clearly that IG can generate "test input" that can be used by the WG to generate the implementation report.

Note: the charter can be "vague" but "input" covers (at least) "test cases/data" (e.g. TD examples) or "test results" (e.g. CSV files declaring the results of tests run externally).

File: wot-ig-2023-draft.html [hunk 2]
```diff
@@ -284,20 +284,25 @@ <h2>Scope</h2>
...
-          <li>test-drive upcoming or proposed technologies for the W3C Recommendation Track</li>
+          <li>test-drive upcoming or proposed technologies for the W3C Recommendation track</li>
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
```
Comment (mmccool) [inline wot-ig-2023-draft.html:301]: probably too much detail...

File: wot-ig-2023-draft.html [hunk 3]
```diff
@@ -284,20 +284,25 @@ <h2>Scope</h2>
...
         <p>
           An important activity is the operation of PlugFests to collect interoperability implementation experience 
-          and to validate the current working assumptions of the potential building blocks discussed in the Web of Things Interest Group.
+          and to validate the current working assumptions of the potential building blocks and features discussed in the Web of Things Working Group task forces (e.g., Thing Description).
           These events enable developers to get together to test their implementations and facilitates networking between partners and experts.
           The Interest Group will seek to encourage work on open source projects and community evaluation of the Web of Things.
           In more detail, PlugFests allow participants to:
         </p>
         <ul>
-          <li>assist with preparing implementation reports for WoT WG Candidate Recommendations</li>
```
Comment (mmccool) [inline wot-ig-2023-draft.html:293]: this was deleted but really we should change "assist" to "provide test input to the WG for the purpose of preparing implementation reports"

Comment (sebastiankb) [reply wot-ig-2023-draft.html:293]: a new text below explain the role of the IG  and how it support to create test inputs

File: wot-ig-2023-draft.html (hunk 1 above)
Comment (sebastiankb) [reply wot-ig-2023-draft.html]: ok, I added another sentence that IG can provide "test input"

**Review by egekorkan — APPROVED** [review]

File: wot-ig-2023-draft.html [hunk 4]
```diff
@@ -284,20 +284,26 @@ <h2>Scope</h2>
...
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
+	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Intersted Group supports the realization of 
+	 the testing phase and, similar to PlugFest, helps with the inclusion of implementations, e.g. from open source projects and partners. Therefore, the Intersted Group can provide test input instances that
```
Comment (mjkoster) [inline wot-ig-2023-draft.html:303]:
```suggestion
	 the testing phase and, similar to PlugFest, helps with the inclusion of implementations, e.g. from open source projects and partners. Therefore, the Interest Group can provide test input that 
```

File: wot-ig-2023-draft.html [hunk 5]
```diff
@@ -284,20 +284,26 @@ <h2>Scope</h2>
...
+          <li>test-drive upcoming or proposed technologies for the W3C Recommendation track</li>
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
+	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Intersted Group supports the realization of
```
Comment (mjkoster) [inline wot-ig-2023-draft.html:302]:
```suggestion
	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Interest Group supports the realization of 
```

File: wot-ig-2023-draft.html [hunk 6]
```diff
@@ -284,20 +284,26 @@ <h2>Scope</h2>
...
-          <li>test-drive upcoming or proposed technologies for the W3C Recommendation Track</li>
+          <li>test-drive upcoming or proposed technologies for the W3C Recommendation track</li>
           <li>identify and test-drive upcoming or proposed technologies as they apply to the collection of technologies required to address specific industry and user community needs</li>
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Testing (or feature testing) is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation documents.
```
Comment (mjkoster) [inline wot-ig-2023-draft.html:301]:
```suggestion
	 Feature testing is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation Track documents.
```

File: wot-ig-2023-draft.html [hunk 7]
```diff
@@ -284,20 +284,26 @@ <h2>Scope</h2>
...
           <li>take part in interoperability experiments across implementations for ideas at different levels of maturity</li>
           <li>reach out to other communities and new members through an OpenDay for non-members, which usually includes a demo track</li>
           <li>ensure interoperability between WoT technologies and assistive technologies used by people with disabilities</li>
         </ul>
 
+	<p>
+	 Feature testing is a specific requirement of the Web of Things Working Group in order to create an implementation report for the publication of the Recommendation Track documents.
+	 The Web of Things working group will define a set of requirements that must be tested by at least two independent implementations. The Web of Things Interest Group supports the realization of 
+	 the testing phase and, similar to PlugFest, helps with the inclusion of implementations, e.g. from open source projects and partners. Therefore, the Interest Group can provide test input that 
+	 can be used by the Working Group to generate the implementation report.
```
Comment (mjkoster) [inline wot-ig-2023-draft.html:304]:
```suggestion
	 can be used by the Working Group, in addition to other inputs, to generate the implementation report.
```

**Pull request merged**